drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -237,15 +237,35 @@
     background-color: #fff;
 }
 
+@media screen and (max-width: 480px) {
+    .xblock--drag-and-drop .target {
+        overflow: auto;
+        display: block;
+    }
+
+    .xblock--drag-and-drop .target-img {
+        max-width: none;
+    }
+}
+
+
+@media screen and (min-width: 481px) {
+    /* Style for non-mobile */
+    .xblock--drag-and-drop .target-img {
+        width: 100%;
+        max-width: 100%;
+    }
+}
+
 .xblock--drag-and-drop .target-img-wrapper {
     /* This element is required for Firefox due to https://bugzilla.mozilla.org/show_bug.cgi?id=975632 */
     display: table-row;
 }
 
 .xblock--drag-and-drop .target-img {
     display: table-cell;
-    width: 100%;
-    max-width: 100%;
```

Comment:
As noted above, `width` and `max-width` should stay here under default styles, but should be overridden under mobile specific styles.